.travis.yml

```diff
@@ -20,10 +20,13 @@ dart_task:
 # them against each Dart version.
 matrix:
   exclude:
-  - dart: stable
-    dart_task: dartfmt
-  - dart: stable
-    dart_task: dartanalyzer
+    - dart: stable
+      dart_task: dartfmt
+    - dart: stable
+      dart_task: dartanalyzer
+  allow_failures:
```

Comment:
We need to get these tests passing!

Reply:
Do you want me to give fixing these a try, or continue down the list of packages first?